doc/helm/user-guide.md

```diff
@@ -309,6 +309,8 @@ kubectl delete -f deploy/role.yaml
 kubectl delete -f deploy/service_account.yaml
 kubectl delete -f deploy/crds/example.com_nginxes_crd.yaml
 ```
+**NOTE** Additional CR/CRD's can be added to the project by running, for example, the command :`operator-sdk add api --api-version=cache.example.com/v1alpha1 --kind=AppService`
+For more information, refer [cli][addcli] doc.
 
 ## Advanced features
 
```

Comment:
If we create the new api with the chart the watch is using the name of the Kind and note the name of the chart. Following an example:
- Run `$ operator-sdk add api --api-version=cache.example.com/v1alpha1 --kind=AppService  --helm-chart=stable/ngnix` in the Helm Memcached sample
- Then, see that the watch will be with `helm-charts/appservice`, however,  the chart name is `nginx-ingress` which will not allow run the project.

Reply:
@camilamacedo86 Great catch, fixing this.